src/client/app/warehouses/warehouses.route.js

```diff
@@ -0,0 +1,32 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('app.warehouses')
+    .run(appRun);
+
+  appRun.$inject = ['routerHelper'];
+  /* @ngInject */
+  function appRun(routerHelper) {
+    routerHelper.configureStates(getStates());
+  }
+
+  function getStates() {
+    return [
+      {
+        state: 'warehouses',
+        config: {
+          url: '/warehouses',
+          templateUrl: 'app/warehouses/warehouses.html',
+          controller: 'WarehousesController',
+          controllerAs: 'vm',
```

Comment:
let's use controllerAs  more wise. each controller = unique namespace like `vmWarehouse`
can you please refactor all vm's in the project?